tests/mic/test_audio_input.py

```diff
@@ -0,0 +1,23 @@
+"""
+Mock tests for microphone input
+"""
+from unittest.mock import patch
+
+from spokestack.mic.pyaudio import PyAudioMicrophoneInput
+
+
+@patch("spokestack.mic.pyaudio.pyaudio")
+def test_audio_input(mock_class):
+    """ Test for microphone input """
```

Comment:
tests generally don't require docstrings, because they wouldn't be included in the api reference
is black or something requiring docstrings for tests?

Reply:
It's not black. I think it is pylint with a bad configuration so it is inspecting tests.